### PR TITLE
add ast.FloorDiv support

### PR DESCRIPTION
**Pull request by JelleZijlstra**

### CODE REVIEW

**Review by KeithTheEE — APPROVED** [review]
It doesn't address if the floor is carpet, tile, grass, earth, or anything else, so there could be type ambiguity down the line. But this seems like a necessary patch for now and seems to function as needed.

**Comment by JelleZijlstra** [comment]
Maybe we should format it as `__` instead. It doesn't make sense that the floor is sloped, it should be flat.

File: bot/utils/blurple_formatter.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
     ast.Mult: "*",
     ast.Sub: "-",
     ast.Div: "/",
+    ast.FloorDiv: "//",
```
Comment (thatbirdguythatuknownot) [inline bot/utils/blurple_formatter.py:12]: Fix is already included in an open issue (issue #30).

**Review by ShakyaMajumdar — APPROVED** [review]

**Review by Bluenix2 — APPROVED** [review]
Thank you for this Jelle, as stable as my actual floor.

**Review by wookie184 — APPROVED** [review]
Seems to work nicely

**Pull request merged**